workflow/tools/util_in_docker/generate_song_payload.py

```diff
@@ -212,6 +218,47 @@ def retrieve_metrics_file(metrics_directory, read_group):
             return file
     raise Exception("The metrics file %s does not exist." % (file_full_path))
 
+def get_workflow_data(wf_name, wf_version, execution_runner_name, execution_runner_version, execution_job_id,yaml_data):
+    return {
+        'name': wf_name,
+        'version': wf_version,
+        'execution': {
+            'runner_name': execution_runner_name,
+            'runner_version': execution_runner_version,
+            'job_id': execution_job_id
+        },
+        'input': get_workflow_data_files(yaml_data)
+    }
+
+def get_workflow_data_files(yaml_data):
+    files = []
+    filename_keys = []
+    for read_group in yaml_data.get('readGroups'):
+        for file in read_group.get('files'):
+            if "song://" in file.get('fileName'):
+                repository = "collaboratory"
```

Comment:
should use `path`, eg, `song://collaboratory/EGAZ00001018588/7aba9867-ffba-5e57-96d0-300c80839504`, something like below:

`_, _, repository, bundle_id, object_id = file.get('path').split('/')`

This way, it will work for repository being `aws` as well, like `song://aws/EGAZ00001018588/7aba9867-ffba-5e57-96d0-300c80839504`, no extra work needed.